modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/04_sales/page-definition.json

```
@@ -396,1522 +552,135 @@
-																												"fi_FI": "View Dashboard",
-																												"fr_FR": "View Dashboard",
-																												"hu_HU": "View Dashboard",
-																												"ja_JP": "View Dashboard",
-																												"nl_NL": "View Dashboard",
-																												"pt_BR": "View Dashboard",
-																												"sv_SE": "View Dashboard",
-																												"zh_CN": "View Dashboard"
-																											}
-																										}
-																									}
-																								}
-																							],
-																							"fragmentStyle": {
-																								"borderRadius": "",
-																								"fontWeight": "fontWeightBold",
-																								"height": "24px",
-																								"paddingRight": "1",
-																								"shadow": "",
-																								"textAlign": "",
-																								"textColor": "colorBrandPrimary"
-																							},
-																							"fragmentViewports": [
-																								{
-																									"fragmentViewportStyle": {
-																										"marginRight": "0"
-																									},
-																									"id": "tablet"
-																								}
-																							],
-																							"indexed": true,
-																							"name": "Card Link"
-																						},
-																						"type": "Fragment"
-																					},
-																					{
-																						"definition": {
-																							"fragment": {
-																								"key": "clay-icon-link"
-																							},
-																							"fragmentConfig": {
-																								"iconLink": "https://www.liferay.com/pt/home",
-																								"iconName": "order-arrow-right"
-																							},
-																							"fragmentFields": [
-																							],
-																							"fragmentStyle": {
-																								"textAlign": "",
-																								"textColor": "linkColor"
-																							},
-																							"indexed": true
-																						},
-																						"type": "Fragment"
-																					}
-																				],
-																				"type": "Section"
-																			}
-																		],
-																		"type": "Section"
+																				{
+																					"fieldName": "privatePage",
+																					"fieldValue": "false"
+																				},
+																				{
+																					"fieldName": "siteKey",
+																					"fieldValue": "PRM Testing Sales2"
+																				}
+																			]
+																		}
 																	}
-																],
-																"type": "Column"
+																}
 															}
-														],
-														"type": "Row"
+														},
+														"text": {
+															"value_i18n": {
+																"en_US": "Register New Deal"
```

```suggestion
																"en_US": "Register New Deal"
```

add another languages